src/alert/interfaces.ts

```diff
@@ -47,6 +47,33 @@ export namespace AlertProps {
   export interface AnalyticsMetadata {
     errorContext?: ErrorContext;
   }
+
+  export interface Style {
```

Comment:
Why style is applying to all types but not to specific type(info, success, error, warning)?